packages/react-composites/review/react-composites.api.md

```diff
@@ -752,6 +752,7 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
 export interface MeetingCompositeStrings {
     chatButtonLabel: string;
     chatPaneTitle: string;
+    newMessage: string;
```

Comment:
Can we call this `newMessageLabel` for consistency with `chatButtonLabel` etc.

Even better would be `newMessageNotificationLabel`.